CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.0.0-dev.1
+- Use camelCase to name internal variables.
```

Comment:
Write the change log from the perspective of our end-users use cases. For example:

- This package now targets `package:lints` for the generated code.
  The generated code uses C symbol names as is.
  Use either `// ignore_for_file: lintRule1, lintRule2` in the preamble, or rename the symbols to make `package:lints` happy.

Which are the most common lint rules that users will have to ignore in the preamble?

Reply:
Updated changelog.

> Which are the most common lint rules that users will have to ignore in the preamble?

Considering how C libraries are: 
- `camel_case_types, non_constant_identifier_names`. (Most libraries use underscores rather than camel case.)
- `unused_element`. (For C declarations starting with `_`)